AppCenterIdentity/AppCenterIdentity/MSIdentity.m

```diff
@@ -131,26 +144,132 @@ + (void)handleUrlResponse:(NSURL *)url {
   [MSALPublicClientApplication handleMSALResponse:url];
 }
 
-- (void)acquireToken {
-  if (self.clientApplication == nil) {
-    return;
++ (void)login {
+
+  // TODO protect with canBeUsed.
```

Comment:
If it's disabled it will prompt an error and will be no op.

Reply:
This should be part of `login` implementation so we just added a reminder there.

Reply:
Should clarify the reminder, I don't understand it.

Reply:
`canBeUsed` is a property that base service has and is used to determine whether the service is available now or not. The TODO means that we should check `canBeUsed` property before calling instance method to make sure it is ready for use. This is a pattern that all other services do. Only reason that we didn't address it and put TODO is we are not implementing `login` method right now. It will be coming separately.

Reply:
ok thanks